lib/bamboo/adapters/send_grid_adapter.ex

```diff
@@ -23,36 +23,23 @@ defmodule Bamboo.SendGridAdapter do
       end
   """
 
+  @service_name "SendGrid"
   @default_base_uri "https://api.sendgrid.com/api"
   @send_message_path "/mail.send.json"
+  @extra_api_error_message """
+  If you are deploying to Heroku and using ENV variables to handle your API key,
+  you will need to explicitly export the variables so they are available at compile time.
+  Add the following configuration to your elixir_buildpack.config:
+
+  config_vars_to_export=(
+    DATABASE_URL
+    SENDGRID_API_KEY
+  )
+  """
```

Comment:
Could you remove these messages as they are no longer necessary (see PR #275 for more info)